website/profile/views.py

```diff
@@ -822,3 +822,14 @@ def redirect_to_twitter(twitter_handle):
         })
 
     return redirect(user.url)
+
+
+def view_institution(**kwargs):
+    inst = Institution.load(kwargs.get('id'))
```

Comment:
As per COSDev, prefer to put path variables directly in the signature: `view_institution(id, **kwargs)` rather than pulling from kwargs. If you make that change, you might want to rename `id` to avoid shadowing the Python built-in.